src/commands/Trivia.js

```diff
@@ -0,0 +1,29 @@
+const { Command } = require('@dyno.gg/dyno-core');
+const superagent = require('superagent');
+
+class Trivia extends Command {
+	constructor(...args) {
+		super(...args);
+
+		this.aliases        = ['trivia']; 
+		this.module         = 'Fun';
+		this.description    = 'Trivia time, get some friends and get a trivia game started! ';
+		this.usage          = 'trivia';
+		this.example        = 'trivia';
+		this.cooldown       = 5000;
+		this.expectedArgs   = 0;
+	}	
+
+		async execute({ message }) {
+			try {
+				let res = await superagent.get('http://jservice.io/api/random'); // http://jservice.io/api/random is the api site, there is so much to add.  
+				set("Accept", "application/json")

```

Comment:
wait be careful here.
You are using `.set()` wich is a method of superagent. but
1) you forgot the `.`
2) you added `;` at the end of the .get request. Wich mean the command is over. This would just error.

Also As mentionned before I suggest at least that you set up a timeout (maybe 30 sec) and then you send a second message with the answer. Otherwise you have the question but never the answer.
This might not be the best way to do a trivia command but at least it would makes more sense :)
here doc for setTimeout() https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout

Reply:
is this still a thing or do I have to remove the .set